site/content/guides/claude-code-monitoring.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Add in Depth Monitoring With Checkly and Claude Code
+description: >-
+  By integrating AI into your development workflow, you can generate valid Checkly constructs—including URL monitors and Playwright-based browser checks—with minimal effort.
+author: Nočnica Mellifera
+avatar: 'images/avatars/nica-mellifera.png'
+tags:
+  - FAQ
```

Comment:
Can we just go with `AI`? For whatever reason, this renders super strange in the guide section.

<img width="792" height="306" alt="grafik" src="https://github.com/user-attachments/assets/49522d2b-27de-4729-860b-7fae6dff31a8" />

Reply:
FAQ AI 😆 fixed